recipes/pyomo.solvers/meta.yaml

```diff
@@ -1,9 +1,9 @@
-# Do metapackages have a version?
-# Do metapackages have a license?
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
We discourage comments on recipes unless they are explaining a patch or referencing that needs to be addressed.